src/mscorlib/src/System/Reflection/RuntimeAssembly.cs

```diff
@@ -800,5 +801,83 @@ public override Module[] GetLoadedModules(bool getResourceModules)
 
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
         internal static extern int GetToken(RuntimeAssembly assembly);
+
+        public sealed override Type[] GetForwardedTypes()
+        {
+            List<Type> types = new List<Type>();
+            List<Exception> exceptions = new List<Exception>();
```

Comment:
It didn't seem worth the complexity given that we have two append points (one in a separate method.)